### ISSUE TITLE
Having 1000 or more cards in your deck causes your deck size to not show the last digit (EX: a deck of 1400 cards will show as a deck of 140)

### ISSUE THREAD

**Issue by datrobino** [PennyDreadfulMTG/modo-bugs#1007]
**What is the expected behaviour?**
 - If you have 1400 card deck your library size will show to be 1400

**What is the actual behaviour?**
 - If you have a 1400 card deck your library will show to be 140

**Screenshot**
![image](https://user-images.githubusercontent.com/85389721/153777814-8e1154e7-c937-47a3-beae-dfda9d5c0397.png)

Affects:

**Comment by Rowsol** [comment]
This problem has been around since v4 launched. I used to love playing my 1000 card deck. Now it you cast a cultivate prepare to have the program stall for 15 seconds.

v2 ran even smoother with big decks. There was a guy who routinely played a 900 card deck in extended games. Everyone was always impressed with it's performance when I played with him. It's ultimately what inspired me to start making big decks.

If only you knew the "golden age" while it was. I would've played a lot more games.
